doc/howto/move_instances.md

```diff
@@ -59,6 +59,12 @@ To do so, ensure the following configuration:
 * Set {config:option}`instance-migration:migration.stateful` to `true` on the instance.
 * Set [`size.state`](devices-disk) of the virtual machine's root disk device to at least the size of the virtual machine's {config:option}`instance-resource-limits:limits.memory` setting.
 
+However, if you create an instance on a shared storage backend like Ceph RBD, you don't have any parameters to set to enable the live migration. This is due to the fact that when the instance is created, it is by default set with {config:option}`instance-migration:migration.stateful` to `true` with the [`size.state`](devices-disk) parameter being unset a this is not strictly required for shared storage.
+
+This new default value for {config:option}`instance-migration:migration.stateful` only applies for new instances. Your existing instances won't be affected by this change.
```

Comment:
```suggestion
The default value for {config:option}`instance-migration:migration.stateful` was introduced with LXD 5.21 and applies only to new instances. Existing instances won't be affected by this change.
```

Remember that these are docs and not release notes, and nobody will know what "new" means. ;)